### ISSUE TITLE
Squiz.WhiteSpace.ScopeKeywordSpacing not respect `final` keyword

### ISSUE THREAD

**Issue by evilebottnawi** [squizlabs/PHP_CodeSniffer#1312]
Not error if i use:
```php
class MyClass
{
    final  public function test() // Here two spaces after final keywork
    {
        return 'foo-bar';
    }
}
```
But `final` not scope, it is keyword. Maybe related to another sniff.

**Comment by aik099** [comment]
That sniff only checks space between of scope and `function` word.

Using that logic there should different sniff that check spacing for `final` and `abstract` keywords in function declaration. I have no if there are.

**Comment by aik099** [comment]
There is `PSR2.Methods.MethodDeclaration` sniff that does something with `final` word, but it does a lot of other stuff as well.

**Comment by evilebottnawi** [comment]
@aik099 maybe new sniff?

**Comment by aik099** [comment]
Sure. Something that check keyword spacing before function keyword. And their order maybe.

**Comment by gsherwood** [comment]
Closing as I've had no additional interest or PRs for this.

**Comment by jrfnl** [comment]
I'm actually interested in a sniff like this - which would check all keywords - and am willing to create it.
Should be easy enough as most of the logic is already available in the aforementioned sniff.
